File: examples/docs-code-snippets/src/main/java/examples/java/recipes/RecipeBlockInteraction.java
package examples.java.recipes;

import com.ing.baker.recipe.javadsl.InteractionFailureStrategy;
import com.ing.baker.recipe.javadsl.Recipe;

public class RecipeBlockInteraction {

    public final static Recipe recipe = new Recipe("example")
        .withDefaultFailureStrategy(
            InteractionFailureStrategy.BlockInteraction()
        );
}

File: examples/docs-code-snippets/src/main/scala/examples/scala/recipes/RecipeBlockInteraction.scala
package examples.scala.recipes

import com.ing.baker.recipe.common.InteractionFailureStrategy
import com.ing.baker.recipe.scaladsl.Recipe

object RecipeBlockInteraction {
  val recipe: Recipe = Recipe("example")
    .withDefaultFailureStrategy(
      InteractionFailureStrategy.BlockInteraction()
    )
}